servicecontrol/creating-config-file.md

```diff
@@ -320,6 +319,118 @@ This entry should be set to `false` if there is no external process reading mess
 
 See [Installation](installation.md) for details on how to set this at install time.
 
+## Usage Reporting
+
+### LicensingComponent/ServiceControlThroughputDataQueue
+
+The queue on which throughput data is received by the ServiceControl Error instance. This setting must match the equivalent [`Monitoring/ServiceControlThroughputDataQueue`](/servicecontrol/monitoring-instances/installation/creating-config-file.md#usage-reporting-monitoringservicecontrolthroughputdataqueue) setting for the Monitoring instance.
+
+In most instances these settings do not need to be modified.
+
+If running multiple setups of the Platform Tools (i.e. multiple versions of ServiceControl Error and Monitoring instances) then modify these settings so that the queue on each Monitoring instance is matched to the queue of its Error instance.
+
+If using [MSMQ transport](/transports/msmq) and the Monitoring instance is installed on a different machine to the ServiceControl Error instance, then only the Monitoring instance setting needs to be modified to include the machine name of the Error instance in the queue address.
+
+Type: string
+
+Default: `ServiceControl.ThroughputData`
+
+### LicenseComponent/ASB/ServiceBusName
+
+When using [Azure Service Bus transport](/transports/azure-service-bus/), this is the Azure Service Bus namespace.
+
+Type: string
+
+### LicenseComponent/ASB/ClientId
+
+When using [Azure Service Bus transport](/transports/azure-service-bus), this is the ClientId for an Azure login that has access to read metrics data for the Azure Service Bus namespace.
+
+Type: string
+
+### LicenseComponent/ASB/ClientSecret
+
+When using [Azure Service Bus transport](/transports/azure-service-bus), this is the ClientSecret for an Azure login that has access to read metrics data for the Azure Service Bus namespace.
+
+Type: string
+
+### LicenseComponent/ASB/TenantId
+
+When using [Azure Service Bus transport](/transports/azure-service-bus), this is the Azure Microsoft Extra ID.
```

Comment:
I'll stop after this one in case you don't agree with the suggestion but you get the idea.

```suggestion
When using the [Azure Service Bus transport](/transports/azure-service-bus), this is the Azure Microsoft Extra ID.
```